Users/develop

sdtsnake opened the pull request:
se termina el crud del users aun falta incluir spring security

sdtsnake committed:
contruccion parcial de clase para el modelo users

se crear las clases parciales para el manejor del
modelo de users pero se hacer commit para hacer
marge de del remoto por cambios del PO

sdtsnake committed:
Merge branch 'main' into users/develop

# Conflicts:
#	src/main/java/work/appdeploys/equipmentControlSystem/exceptions/UsersExceptionBadRequest.java
#	src/main/java/work/appdeploys/equipmentControlSystem/exceptions/UsersExceptionInternalError.java

sdtsnake committed:
se termina el crud del modelo de usuarios

se termina el modelo de usuarios basico falta spring security

sdtsnake committed:
mejora segun el sonar

segun el sonar a mi entender cree que mi expresion
regular es una contrañsea

jeslava10 commented:
Se debe ajustar esta advertencias del sonar 

![image](https://user-images.githubusercontent.com/15691593/188492914-0c6bd6d4-18ca-4bf7-bee6-e8f6deeeb675.png)

jeslava10 commented:
Añadir el regex a al archivo propety  y cargar la variables con regex usando el  @value  https://www.arquitecturajava.com/spring-boot-properties-y-value/ o  usar environment.getProperty("nombre porpiedad") para asignar el valor

sdtsnake committed:
se cambia la manera de estblacer las constantes

segun el sonar se debe cambiar la manera de establecer las constastes
de la expresiones regulares y se dejan en el aplication propierties
para usar la notacion @Value de spring

jeslava10 commented:
imagen de la solucion 
![image](https://user-images.githubusercontent.com/15691593/188743983-d40a4e2e-c41a-4e39-9c5b-a31474240f49.png)

sdtsnake committed:
mejoras segun el sonar

sdtsnake committed:
cambio de los paquetes de camell a normal segun sonar
